dex/networks/bch/params.go

```diff
@@ -0,0 +1,51 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org
+
+package bch
+
+import (
+	"decred.org/dcrdex/dex/networks/btc"
+	"github.com/btcsuite/btcd/chaincfg"
+	bchchaincfg "github.com/gcash/bchd/chaincfg"
+)
+
+var (
+	// MainNetParams are the clone parameters for mainnet.
+	MainNetParams = btc.ReadCloneParams(&btc.CloneParams{
+		PubKeyHashAddrID: bchchaincfg.MainNetParams.LegacyPubKeyHashAddrID,
+		ScriptHashAddrID: bchchaincfg.MainNetParams.LegacyScriptHashAddrID,
+		Bech32HRPSegwit:  "bitcoincash",
+		CoinbaseMaturity: 100,
+		// BitcoinCash didn't change any of these net identifiers, so these are
+		// all spoofed. The only place we currently use Net is in
+		// btcd/chaincfg.Register, where it is checked to prevent duplicate
+		// registration, so our only requirement is that it is unique. This one
+		// was just generated with a prng.
+		Net: 0xe8f3e1e3,
+	})
+	// TestNet3Params are the clone parameters for testnet.
+	TestNet3Params = btc.ReadCloneParams(&btc.CloneParams{
+		PubKeyHashAddrID: bchchaincfg.TestNet3Params.LegacyPubKeyHashAddrID,
+		ScriptHashAddrID: bchchaincfg.TestNet3Params.LegacyScriptHashAddrID,
+		Bech32HRPSegwit:  "bchtest",
+		CoinbaseMaturity: 100,
```

Comment:
These 4 appear same on testnet4, https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/blob/a5e1d83f308f0428b401816b113e9cf3fd1ecb03/src/chainparams.cpp#L499-500 (6f and c4)